sample_file.py

```
@@ -0,0 +1,47 @@
+USERNAME = "admin2"
+PASSWORD = "password123"
+
+import os
+
+result = 0
+
+def infinite_loop():
```

Ruff

Missing return type annotation for public function `infinite_loop`

Add return type annotation: `None`

(ANN201)